### ISSUE TITLE
[feature] Labels for % sliders

### ISSUE THREAD

**Issue by herbertkb** [smrowley/retire-calc#1]
As a user, I  need to know what those sliders mean so I can better understand the UI and plan for retirement.

**Comment by smrowley** [comment]
Agreed. I will work on fixing this. Also, do you have any recommendations over the slider? John has a distaste for them, so I would like something that would be easier to navigate.
